python/servo/testing_commands.py

```diff
@@ -585,7 +585,7 @@ def test_android_startup(self, release, dev):
         process = subprocess.Popen(args, stdout=subprocess.PIPE)
         try:
             while 1:
-                line = process.stdout.readline()
+                line = process.stdout.readline().decode("utf-8")
```

Comment:
This might still fail if the line happens to not be UTF-8. Instead we could stay with bytes all the way, and replace `print(line.rstrip())` below with `sys.stdout.write(line)`.

Reply:
what about the test? `if "JavaScript is running!" in line` fails.

Reply:
As in, the `if` line fails with a UnicodeDecodeError error.